app/components/work_file_list_show_component.rb

```diff
@@ -30,7 +30,7 @@ def member_list_for_display
       end
 
       # omit "portrait" role
-      members = members.find_all {|m| ! m.role_portrait? }
+      members = members.find_all {|m| !(m.asset? && m.role_portrait?) }
```

Comment:
A slightly more fluent way to ssay this might be `!(m.respond_to?(:role_portrait?) && m.role_portrait?)

OR, rails `try` can do this for you -- just `!m.try(:role_portrait?)`, if there is no method it will return `nil` which is false, `!nil` is true, good. 

Same for below.